zcash_client_sqlite/src/wallet.rs

```diff
@@ -233,10 +233,10 @@ impl ViewingKey {
         }
     }
 
-    fn uivk_str(&self, params: &impl Parameters) -> Result<String, SqliteClientError> {
+    fn uivk(&self) -> UnifiedIncomingViewingKey {
```

Comment:
```suggestion
    pub fn uivk(&self) -> UnifiedIncomingViewingKey {
```
(with changelog entry).